docs: Update README (#1127)
thm-mni-ii/feedbacksystem: README.md

## Install

### Development

The following software is required for the development:
    
- Java (Version 11)
- Docker
- Git

Clone this Repository to your locale Directory
```
git clone git@github.com:thm-mni-ii/feedbacksystem.git
```

Change to the cloned directory
```
cd feedbacksystem
```

Build all container an start them with `docker compose`
```
docker compose up -d --build
```

The System can now be accessed at `https://localhost`.

#### Frontend

For frontend development the following software is needed:

- node
- npm

Change to the Directory of the frontend code
```
cd modules/fbs-core/web
```

Install neccessary npm packages and start the dev-server
```
npm i
npm run start
```

### Production

#### Requirements

* A Kubernetes Cluster
* [kubectl](https://kubernetes.io/docs/tasks/tools/)
* [helm](https://helm.sh/docs/intro/install/)
* [deno](https://deno.land/manual/getting_started/installation)

#### Steps

1. Ensure the requirements are met
3. Generate values 
```

> deno run --reload=https://raw.githubusercontent.com https://raw.githubusercontent.com/thm-mni-ii/feedbacksystem/dev/chart/generate-values.ts --allow-write=vals.yaml vals.yaml

Reviewer: ```suggestion
deno run --reload=https://raw.githubusercontent.com https://raw.githubusercontent.com/thm-mni-ii/helm-charts/main/charts/feedbacksystem/generate-values.ts --allow-write=vals.yaml vals.yaml
```

```
4. Add the helm repository 
```

> helm repo add feedbackssystem <url>

Reviewer: ```suggestion
helm repo add thm-mni-ii https://thm-mni-ii.github.io/helm-charts
```

```
5. Install 
```

> helm install -n <namepsace> --create-namespace --wait -f vals.yaml fbs feedbackssystem/feedbackssystem

Reviewer: ```suggestion
helm install -n <namepsace> --create-namespace --wait -f vals.yaml fbs thm-mni-ii/feedbackssystem
```

```

## API

> The specification of the interfaces of the feedbacksystem can be found under the following link:
> 
> [https://github.com/thm-mni-ii/feedbacksystem/blob/dev/modules/fbs-core/api/api-docs.yml](https://github.com/thm-mni-ii/feedbacksystem/blob/dev/modules/fbs-core/api/api-docs.yml)

Reviewer: ```suggestion
The specification of the interfaces of the feedbacksystem can be found [here](https://thm-mni-ii.github.io/feedbacksystem/api-docs/).
```
